pkgs/by-name/xf/xfox/package.nix

```
@@ -0,0 +1,32 @@
+{
+  python3Packages,
+  fetchFromGitHub,
+  lib,
+}:
+python3Packages.buildPythonApplication rec {
+  pname = "xfox";
+  version = "1.0.0";
+  pyproject = true;
+  src = fetchFromGitHub rec {
+    owner = "pxlman";
+    repo = "xfox";
+    tag = "v${version}";
+    sha256 = "sha256-gbcJ02ZaPqiCo0VHnVOYzrWR00dQXgKavFTgP+6rS34=";
+  };
+  propagatedBuildInputs = [
+    python3Packages.evdev
+    python3Packages.libevdev
+    python3Packages.setuptools
+  ];
+  postPatch = ''
+    substituteInPlace setup.py --replace "evdev-binary" "evdev"
```

```suggestion
    substituteInPlace setup.py --replace-fail "evdev-binary" "evdev"
```
Plain `--replace` is deprecated and warns when used iirc. `--replace-fail` is safer since you'll know something changed here, and won't just quietly fail